scripts/source-maintenance.sh

```diff
@@ -232,10 +246,10 @@ for FILENAME in `git ls-files`; do
 	chmod 755 ${FILENAME}
 	;;
 
-    Makefile.am)
-
-    	perl -p -e 's/@([A-Z0-9_]+)@/\$($1)/g' <${FILENAME} >${FILENAME}.styled
-	mv ${FILENAME}.styled ${FILENAME}
+    *Makefile.am)
```

Comment:
I also found doc/manuals/Substitute.am, src/Common.am, and src/TestHeaders.am. AFAICT, the new script should be applied to (at least the last two of) those files even if it does not change them.

If you can go by the `.am` extension, let's do that.

Reply:
Going for .am extension